test/cpp/interop/stress_test.cc

```diff
@@ -36,9 +36,7 @@
 #include "test/cpp/util/metrics_server.h"
 #include "test/cpp/util/test_config.h"
 
-extern "C" {
 extern void gpr_default_log(gpr_log_func_args* args);
```

Comment:
Why is this line even needed? You're already including the log header file.

Reply:
Once you make this change, I'll approve.

Reply:
So this is actually needed. the header file doesn't declare gpr_default_log, that is kept in the .cc file. And stress test wraps that function, so it needs this extern decl

Reply:
Ok, I get it.